### PR TITLE
BAU: Show more details about certificates for admins

### PR DESCRIPTION
**Pull request by jakubmiarka**
The admin interface was quite limited in what it was showing about the certificates attached
to a component. This change exposes the common name, expiry date and the certificate value.
This should make it easier to see what certs exactly are in use. I also restructured the page and it doesn't show the disabled certs by default, but are collapsed.

Also, simplified the partial views to use only one and removed a test as it was
no longer possible to test an invalid test. However, this logic is covered within the
event validation and should not be tested via system tests anyway.

**Before:**
image

**After:**
image

### CODE REVIEW

File: app/views/certificates/status/_certificate_table.html.erb [hunk 1]
```diff
@@ -2,36 +2,55 @@
...
   <tbody class="govuk-table__body">
    <% certificates.each do |certificate| %>
       <tr class="govuk-table__row" id="certificate_table_<%= certificate.id %>">
-        <th class="govuk-table__header" scope="row"><%= format_date_time(certificate.updated_at) %></th>
-        <td class="govuk-table__cell"><%= certificate.id %></td>
+        <td class="govuk-table__cell">
+          ID: <%= certificate.id %> <br>
+          Subject: <strong><%= certificate.x509.subject %></strong> <br>
+          Expires: <%= certificate.x509.not_after %> <br>
+          Value:
```
Comment (AmritSidhu) [inline app/views/certificates/status/_certificate_table.html.erb]: I know this is admin so doesn't particularly matter, but for consistency with the rest of the file should these be localised

**Comment by AmritSidhu** [comment]
Do we really need to show the enabled or disabled column for the signing certificates considering its in the title. Just less unnecessary information on the page

**Comment by jakubmiarka** [comment]
> Do we really need to show the enabled or disabled column for the signing certificates considering its in the title. Just less unnecessary information on the page

Yeah, good point. It's only useful in the All certificates section I guess.... I'll drop it then, maybe even the Usage then?

File: app/views/certificates/status/_certificate_table.html.erb (hunk 1 above)
Comment (jakubmiarka) [reply app/views/certificates/status/_certificate_table.html.erb]: yep can do

Comment (jakubmiarka) [reply app/views/certificates/status/_certificate_table.html.erb]: updated

**Pull request merged**